### PR TITLE
Prevent force overridden of PS1 if already set (release-1.1)

### PR DESCRIPTION
**Pull request by panda1100**
This cherry-picks #924 #929 into the release-1.1 branch.

Signed-off-by: Yoshiaki Senda <yoshiaki@live.it>

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/apptainer/apptainer/blob/main/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/apptainer/apptainer/blob/main/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/apptainer/apptainer/blob/main/CONTRIBUTING.md)
- Added tests to validate this PR, linted with `make check`  and tested this PR locally with a `make test`, and `make testall` if possible (see CONTRIBUTING.md).
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/apptainer/apptainer/blob/main/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/apptainer/apptainer/blob/main/CONTRIBUTORS.md)

### CODE REVIEW

**Comment by DrDaveD** [comment]
Oh, I thought this would wait until release 1.2.0.  I'm not sure when or if there's going to be a 1.1.6.  If you would like it to be slated for 1.1.6 anyway, could you come to a community meeting?  Or otherwise perhaps bring it up on the #devel slack channel to be discussed there.

**Comment by panda1100** [comment]
> If you would like it to be slated for 1.1.6 anyway, could you come to a community meeting?
@DrDaveD I would be happy to join the upcoming meeting on Tue 17th Jan 8:00am PST!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ The Singularity Project has been
 and re-branded as Apptainer.
 For older changes see the [archived Singularity change log](https://github.com/apptainer/singularity/blob/release-3.8/CHANGELOG.md).
 
+## Changes Since Last Release
+
+- Make `PS1` envrionment variable changeable via `%environment` section on
```
Comment (DrDaveD) [inline CHANGELOG.md:10]: Fix typo, `envrionment` => `environment`.

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (panda1100) [reply CHANGELOG.md]: Thank you @DrDaveD -san! I fixed typo.

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**